### PR TITLE
add qt6 apt package suggestion

### PR DESCRIPTION
**Pull request by trim21**

### CODE REVIEW

File: Compilation-Debian-and-Ubuntu.md [hunk 1]
```diff
@@ -34,10 +35,16 @@ qBittorrent uses the Qt framework as the basis for its GUI.
 
 qBittorrent 4.0 - 4.1.x requires at least Qt 5.5.1, and qBittorrent 4.2 and later requires at least Qt 5.9.
 
-For Debian 10, Ubuntu 18.04 LTS or later, just install Qt from the official repositories:
+For Debian 10, Ubuntu 18.04 LTS or later, if you build qBittorrent with Qt5 just install Qt from the official repositories:
 
 ```bash
-sudo apt install qtbase5-dev qttools5-dev libqt5svg5-dev
+sudo apt install qtbase5-dev qttools5-dev libqt5svg5-dev qtbase5-private-dev
```
Comment (Chocobo1) [inline Compilation-Debian-and-Ubuntu.md:41]:
```suggestion
sudo apt install qtbase5-dev qtbase5-private-dev qttools5-dev libqt5svg5-dev
```

**Comment by Chocobo1** [comment]
@trim21 
Thank you!

**Comment by Piccirello** [comment]
It's worth noting that even on Ubuntu 24.04, the [qt6 packages](https://packages.ubuntu.com/search?keywords=qt6-base-dev&searchon=names&suite=noble&section=all) are too old to be used. Ubuntu supplies qt 6.4.2 while qBittorrent requires 6.5.0. I wish qBittorrent's version requirements would take into account Ubuntu's available packages.

**Comment by trim21** [comment]
> It's worth noting that even on Ubuntu 24.04, the [qt6 packages](https://packages.ubuntu.com/search?keywords=qt6-base-dev&searchon=names&suite=noble&section=all) are too old to be used. Ubuntu supplies qt 6.4.2 while qBittorrent requires 6.5.0. I wish qBittorrent's version requirements would take into account Ubuntu's available packages.

It means qbittorent didn't check qt version it's using and just compile?

**Comment by Piccirello** [comment]
I'd have to try tonight/tomorrow to confirm but I would expect it to fail on the master branch.

https://github.com/qbittorrent/qBittorrent/blob/dd34c85884c77e3d4d6fc7223decdf47789affb6/CMakeLists.txt#L11

**Comment by trim21** [comment]
looks like apt packages inside docker and outside docker have different versions.

for example, boost inside docker is 1.74 but 1.83 in docker image ubunto:24.04.

**Comment by trim21** [comment]
interesting， looks like I didnt actually use master branch, because it can't compile it anymore in ci.

**Pull request merged**